app/views/thread.js

```diff
@@ -145,6 +113,10 @@
             const ids = await this.model.getMembers();
             const users = await F.atlas.getContacts(ids);
             const members = [];
+            if (!this.model.get('notices')) {
+                this.model.set('notices', []);
```

Comment:
This is a bit suspect given that were in a view and not the model.   It would probably be better to leave it alone when not actually making alterations and handle the condition of it just being falsy in some settings.

Reply:
I'm doing this because in a fresh thread, it won't open up the side bar since because in the code below it where !!notices.length is null since it was just created. That's why I first check if it exists and if it doesnt, then create an empty array so it has a length of 0.